src/main/scala/org/broadinstitute/hail/driver/LogisticRegressionCommand.scala

```diff
@@ -15,13 +15,13 @@ object LogisticRegressionCommand extends Command {
   def description = "Test each variant for association using the logistic regression model"
 
   class Options extends BaseOptions {
-    @Args4jOption(required = true, name = "-t", aliases = Array("--test"), metaVar = "TEST",  usage = "Statistical test: `wald', `lrt', or `score'")
+    @Args4jOption(required = true, name = "-t", aliases = Array("--test"), usage = "Statistical test: `wald', `lrt', or `score'")
```

Comment:
At first I started adding more metaVars, and then I I decided I wanted to be consistent on usage across the the three regression models and wasn't yet sure of plan so decided to rip out all metaVars for now and then come back in another PR across all three.